sdk/go/dag/dag.gen.go

```diff
@@ -97,6 +97,8 @@ func Engine() *dagger.Engine {
 }
 
 // Initialize a new environment
+//
+// Experimental: "Environments are not yet stabilized"
```

Comment:
These quotes seem a bit off

Reply:
Oops, that was a go specific problem, have updated there :tada: